packages/yew-bootstrap/src/component/pagination.rs

```diff
@@ -0,0 +1,95 @@
+use yew::prelude::*;
+
+
+#[derive(Clone, PartialEq, Eq)]
+pub enum PaginationSize {
+    Large,
+    Normal,
+    Small,
+}
+
+impl Default for PaginationSize {
+    fn default() -> Self {
+        PaginationSize::Normal
+    }
+}
+
+pub struct Pagination {}
+
+#[derive(Properties, Clone, PartialEq)]
+pub struct Props {
+    #[prop_or_default]
+    pub callback: Callback<u32>,
```

Comment:
I would name this prob something more informative in the callback style, `on_click_page` or `on_select_page`. Something like that.